docs/architecture/grpc-for-wcf-developers/kubernetes.md

```diff
@@ -1,30 +1,30 @@
 ---
-title: Kubernetes - gRPC for WCF Developers
+title: Kubernetes - gRPC for WCF developers
 description: Running ASP.NET Core gRPC services in a Kubernetes cluster.
 ms.date: 09/02/2019
 ---
 
 # Kubernetes
 
-Although it's possible to run containers manually on Docker hosts, for reliable production systems it's preferable to use a Container Orchestration Engine to manage multiple instances running across several servers in a cluster. There are various Container Orchestration Engines available, including Kubernetes, Docker Swarm and Apache Mesos. But of these engines, Kubernetes is far and away the most widely used, so it will be the focus of this chapter.
+Although it's possible to run containers manually on Docker hosts, for reliable production systems it's better to use a container orchestration engine to manage multiple instances running across several servers in a cluster. There are various container orchestration engines available, including Kubernetes, Docker Swarm, and Apache Mesos. But of these engines, Kubernetes is far and away the most widely used, so it will be the focus of this chapter.
```

Comment:
I'm really not a fan of the term `container orchestration engines`. I think container orchestrator is a better fit here. Engine seems oddly specific

Reply:
I'm okay not making a change here.